src/schema/rules/sidecars/events.yaml

```diff
@@ -11,3 +11,13 @@ StimulusPresentation:
     - suffix == "events"
   fields:
     StimulusPresentation: recommended
+
+StimulusPresentation_for_eyetracking:
+  selectors:
+    - suffix == "events"
+    - intersects([suffix], ["eyetrack"])
+  fields:
+    StimulusPresentation.ScreenSize: required
+    StimulusPresentation.ScreenResolution: required
+    StimulusPresentation.ScreenDistance: required
+    StimulusPresentation.ScreenRefreshRate: recommended
```

Comment:
Not sure if this is the proper way to set the requirement level of those subfields

Reply:
Agree, written down in the TODO notes of the hackmd.

Reply:
I have added an admonition in the task events section, would that address the issue (in addition to removing this from the schema, as it doesn't have any effect).

Reply:
I think we want to keep that in the schema as the validator will use it for dataset validation.

Reply:
The selectors are definitely off:
```
  selectors:
    - suffix == "events"
    - intersects([suffix], ["eyetrack"])
```

How would you say that this applies to "events" files when there is a "physio" suffix with "PhysioType == eyetrack"?